src/components/FoodBoxList.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Food Box List.
+ */
+
+import React, { Component } from 'react';
+import FoodBox from './FoodBox';
+
+class FoodBoxList extends Component {
```

Comment:
I would personally avoid using this FooBoxListe component just for running a loop, and instead run the map directly in the component before (in this case App.js).